library/ssl_ciphersuites.c

```diff
@@ -395,7 +395,7 @@ static const mbedtls_ssl_ciphersuite_t ciphersuite_definitions[] =
       0 },
 #endif /* MBEDTLS_GCM_C */
 #endif /* MBEDTLS_SHA256_C */
-#if defined(MBEDTLS_SHA512_C)
+#if defined(MBEDTLS_SHA384_C)
```

Comment:
This fixes https://github.com/ARMmbed/mbedtls/issues/4499.

This pull request needs:
* Changelog entries for the fixes of https://github.com/ARMmbed/mbedtls/issues/4472 and https://github.com/ARMmbed/mbedtls/issues/4499.
* Changelog entries for the changes in configuration options.
* A [migration guide entry](https://github.com/ARMmbed/mbedtls/blob/development/docs/3.0-migration-guide.d/00README).